.github/workflows/rust-core.yml

```diff
@@ -29,10 +29,10 @@ jobs:
           target
         key: rust-core-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
-    - name: Install Rust 1.55.0 toolchain
+    - name: Install Rust 1.56.0 toolchain
       uses: actions-rs/toolchain@v1
       with:
-          toolchain: 1.55.0
```

Comment:
Other Github Actions workflows like `Android.yml` also hardcode 1.55.0 and should be adjusted as well, for consistency.

Reply:
Should we also update the 1.55 reference in the README?

Reply:
> Should we also update the 1.55 reference in the README?

Yep that would make sense too, well spotted.

Reply:
Ok, I updated all the references to 1.55